main.gms

```diff
@@ -585,10 +586,9 @@ parameter
 parameter
   c_solscen                 "solar option choice"
 ;
-  c_solscen        = 1;        !! def = 1  !! regexp = [1-3]
+  c_solscen        = 1;        !! def = 1  !! regexp = [1-2]
 *' *  (1): yes
-*' *  (2): no solar
-*' *  (3): fix at bau level
+*' *  (2): fix at bau level
```

Comment:
can be very dangerous, if someone is using old scenario_config files. Keep the number for the same meaning. If you delete a number, we can get gaps but that is o.k.

Reply:
forgot to mention... solscen might be deprecated, no one seems to use it in the current csv configs..